editor/src/java/com/defold/editor/Start.java

```diff
@@ -151,6 +187,30 @@ public void start(Stage primaryStage) throws Exception {
             logger.error("failed start", t);
             throw t;
         }
+
+        try {
+            if (Taskbar.isTaskbarSupported() && Taskbar.getTaskbar().isSupported(Taskbar.Feature.MENU) && isPathsAvailable()) {
+                Taskbar taskbar = Taskbar.getTaskbar();
+                PopupMenu dockMenu = new PopupMenu();
+                MenuItem dockMenuItem = new MenuItem("New window");
```

Comment:
I think it should be "New Window", with a capitalized "W", to match the capitalization of the other entries in the menu?